Technical_Strategy.md

```diff
@@ -17,3 +17,94 @@ A lot of workflow management data in our systems is confidential and needs to be
 *Data in-transit*: All data in transit will be encrypted using TLS 1.2, or higher. This constraint may be relaxed for data movement within a secure firewall with no access to the internet (a VPC for example). 
 
 *Data at-rest*: All data at rest will be encrypted using server-side encryption. We will use customer-managed KMS keys to ensure we have complete control over key creation and management. We’ll perform envelope encryption for encrypting all data at rest.
+
+#### Initial list of AWS Services used
```

Comment:
This is great! Are you planning on adding more sample templates here? If not, I am okay with merging for now

Reply:
We can treat this as a living document, and as we encounter more services or more confidence on things like AppSync/GraphQL API we can add it to that

Reply:
Good idea.